btr-web/btr-main-app/utils/btr-bods/bods-to-si-schema-converters.ts

```diff
@@ -224,7 +224,8 @@ export const getSIsFromBtrBodsSubmission = (submission: BtrFilingI): SiSchemaTyp
     const oocs = _findOwnershipOrControlStatement(submission, person.statementID)
     if (person && oocs) {
       const si = _getSi(person, oocs, businessIdentifier)
-      if (si.effectiveDates.filter(date => !date.endDate).length === 0) {
+      const fullInfo = !(si.couldNotProvideMissingInfo && si.missingInfoReason)
```

Comment:
for full info you can check `!si.couldNotProvideMissingInfo` if that one is provided, the other one is mandatory.